pwiz_tools/Skyline/Model/Export.cs

```diff
@@ -4625,6 +4629,10 @@ protected override void WriteTransition(TextWriter writer,
                 writer.Write(FieldSeparator);
                 writer.WriteDsvField(nodeTranGroup.TransitionGroup.LabelType.ToString(), FieldSeparator);
             }
+            writer.Write(FieldSeparator);
+            writer.Write(nodeTranGroup.PrecursorAdduct.AdductCharge);

```

Comment:
in some formats I've seen, "charge" is an absolute value even for negative data, presumably not the case here

Reply:
In conversation you've told me that this is informational only, so that won't be an issue for interacting with other systems

Reply:
yes, good